### PR TITLE
WanManager support for Turris-omnia

### PR DESCRIPTION
**Pull request by KaviyaKumaresan**
REFPLTB-1505: Add necessary updates for WanManager support in meta-turris layer

Reason for change: Including the changes for wanmanager integeration
Test Procedure: Build and test
Risks: low

Signed-off-by: KaviyaKumaresan <Kaviya.Kumaresan@ltts.com>

### CODE REVIEW

File: conf/machine/turris.conf [hunk 1]
```diff
@@ -44,6 +44,9 @@ require conf/include/turris-bbmasks.inc
 
 DISTRO_FEATURES_append = " meshwifi"
 DISTRO_FEATURES_append = " ipv6"
+DISTRO_FEATURES_append = " rdkb_power_manager"
+DISTRO_FEATURES_append = " rdkb_gpon_manager"
```
Comment (manigandanj) [inline conf/machine/turris.conf:48]: do we need this line?

File: recipes-rdkb/sysint-broadband/sysint-broadband.bbappend [hunk 2]
```diff
@@ -2,7 +2,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRCREV_FORMAT = "${AUTOREV}"
 
-SRC_URI_remove = "${CMF_GIT_ROOT}/rdkb/devices/intel-x86-pc/emulator/sysint;module=.;protocol=${CMF_GIT_PROTOCOL};branch=${CMF_GIT_BRANCH};destsuffix=git/device;name=sysintdevice"
+SRC_URI_remove_dunfell = "${CMF_GIT_ROOT}/rdkb/devices/intel-x86-pc/emulator/sysint;module=.;protocol=${CMF_GIT_PROTOCOL};branch=${CMF_GIT_BRANCH};destsuffix=git/device;name=sysintdevice"
```
Comment (scthunderbolt) [inline recipes-rdkb/sysint-broadband/sysint-broadband.bbappend:5]: Why add the dunfell suffix?

Comment (KaviyaKumaresan) [reply recipes-rdkb/sysint-broadband/sysint-broadband.bbappend]: updated

File: conf/machine/turris.conf (hunk 1 above)
Comment (KaviyaKumaresan) [reply conf/machine/turris.conf]: thats not required, i have removed

**Review by manigandanj — APPROVED** [review]

**Pull request merged**